lib/exception_hunter/config.rb

```diff
@@ -1,7 +1,8 @@
 module ExceptionHunter
   class Config
+    cattr_accessor :admin_user_class,
+                   :current_user_method, :user_attributes
```

Comment:
`current_user_method` is no longer needed right?

Reply:
go home, you're drunk